src/seedu/addressbook/Main.java

```diff
@@ -88,7 +88,8 @@ private void runCommandLoopUntilExitCommand() {
 			do {
 				String userCommandText = ui.getUserCommand();
 				checkFileExist();
-				command = new Parser().parseCommand(userCommandText);
+				new Parser();
+				command = Parser.parseCommand(userCommandText);
```

Comment:
What is the `new Parser()` for? I guess this is for T4A2, where u use Parser as a static class. Try not to have non-meaningful commit names.